unity/src/test/scala/io/delta/unity/UCCatalogManagedClientSuite.scala

```diff
@@ -34,7 +34,23 @@ class UCCatalogManagedClientSuite extends AnyFunSuite with UCCatalogManagedTestU
     }
   }
 
-  // TODO: loadTable throws on invalid input. We need a non-null UCClient in order to test this.
+  test("loadTable throws on invalid input") {
+    val ucClient = new InMemoryUCClient("ucMetastoreId")
+    val ucCatalogManagedClient = new UCCatalogManagedClient(ucClient)
+
+    assertThrows[NullPointerException] {
+      ucCatalogManagedClient.loadTable(null, "ucTableId", "tablePath", 1L)
```

Comment:
nit: consider commenting literal parameters with parameter name?

Reply:
Yes! I'd love to, but our scala formatter complains about that :( You can try it out yourself, it won't let you do `.loadTable(null /* param */, xxx /* param */)`

Reply:
that works for the 1st param (after the `(`) but does not work for other params, (after a `,`)